docs/source/developer_guide/documentation.rst

```diff
@@ -22,15 +22,20 @@ sktime uses the numpydoc_ Sphinx extension and follows
 `NumPy docstring format <https://numpydoc.readthedocs.io/en/latest/format.html>`.
 
 To ensure docstrings meet expectations, sktime uses a combination of validations built into numpydoc_,
-pydocstyle_ (set to the NumPy convention) pre-commit checks, automated testing of docstring examples to ensure
-the code runs without error, and :ref:`reviewer <reviewer_guide_doc>` feedback.
+pydocstyle_ (set to the NumPy convention) pre-commit checks and automated testing of docstring examples to ensure
```

Comment:
```suggestion
pydocstyle_ (set to the NumPy convention), pre-commit checks and automated testing of docstring examples to ensure
```

Reply:
I've accepted most of the other checks, but here I"m referencing pydocstyle as one of the pre-commit checks rather than pydocstyle separate from the pre-commit checks.

Reply:
maybe move "(set to the NumPy convention)" so it is after "pre-commit checks"?

Reply:
I'm fine with that.